src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -12,17 +12,20 @@ public class StandardOutputVisitor : XmlTestExecutionVisitor
         readonly object consoleLock;
         readonly ConcurrentDictionary<string, ExecutionSummary> completionMessages;
         readonly string defaultDirectory;
+        readonly bool showProgress;
 
         public StandardOutputVisitor(object consoleLock,
                                      string defaultDirectory,
                                      XElement assemblyElement,
                                      Func<bool> cancelThunk,
-                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null)
+                                     ConcurrentDictionary<string, ExecutionSummary> completionMessages = null,
+                                     bool showProgress = false)
```

Comment:
I don't have a specific concern, I just wanted to ask the question as I always do when I see public APIs changing like this. I'm OK with this if we don't know of any external consumers of this API, I just wanted to pose the question to ensure it was thought about.